client/src/components/auth/Login.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import axios from 'axios'
+import {Link} from 'react-router-dom'
+
+
+class Login extends Component {
+    state = {
+        username: '',
+        password: ''
+    }
+
+    handleChange = e => {
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    signin = e =>{
+        e.preventDefault()
+
+        axios
+            .post('http://localhost:3300/api/login', this.state)
+            .then(res => {
+                localStorage.setItem('jwt', res.data.token);
+                this.props.history.push('/users')
+            })
+            .catch(err => {
+                console.err(err);
+            });
+    }
+
+    render() {
+        return (
+            <form onSubmit = {this.signin}>
+                <div>
+                    <label>Username</label>
+                    <input name = 'username' value = {this.state.username} onChange = {this.handleChange} type="text"></input>
+                </div>
+                <div>
+                    <label>Password</label>
+                    <input name = 'password' value = {this.state.password} onChange = {this.handleChange} type="password"></input>
+                </div>
+                <div>
+                    <button type = "submit">Sign In</button>
```

Comment:
For me, the Sign In button is not executing the signin function and redirecting me to /users as your code indicates it should.